go/tools/gazelle/merger/fix.go

```diff
@@ -25,7 +26,211 @@ import (
 // FixFile updates rules in oldFile that were generated by an older version of
 // Gazelle to a newer form that can be merged with freshly generated rules.
 func FixFile(oldFile *bf.File) *bf.File {
-	return fixLoads(oldFile)
+	fixedFile := squashCgoLibrary(oldFile)
+	return fixLoads(fixedFile)
```

Comment:
As we add more fixers in here, order is going to be very important (and fragile), might be worth adding comments about which things must happen before others and why (like fixLoads is probably last, because other rules might change what needs loading)